torch/nn/parameter.py

```diff
@@ -1,11 +1,11 @@
 from collections import OrderedDict
 
 import torch
-from torch._C import _disabled_torch_function_impl
+from torch._C import _disabled_torch_function_impl, _TensorMeta
 
 
 # Metaclass to combine _TensorMeta and the instance check override for Parameter.
-class _ParameterMeta(torch._C._TensorMeta):
```

Comment:
Can you clarify why we're doing such changes? If we don't have a linter to enforce this kind of stylistic style preference, I don't think we should change it.